.travis.yml

```diff
@@ -1,11 +1,15 @@
 language: ruby
 
 rvm:
-- 2.3
-- 2.4
```

Comment:
I dropped these Ruby version because they don't work with Rails 4.x.
If you want to keep them, I'll update this PR.